semantic-python/test/Test.hs

```diff
@@ -69,15 +72,20 @@ assertJQExpressionSucceeds directive tree core = do
   catch @_ @Streaming.Process.ProcessExitedUnsuccessfully jqPipeline $ \err -> do
     HUnit.assertFailure (unlines [errorMsg, dirMsg, jsonMsg, astMsg, treeMsg, treeMsg', show err])
 
-fixtureTestTreeForFile :: HasCallStack => FilePath -> Tasty.TestTree
-fixtureTestTreeForFile fp = HUnit.testCaseSteps fp $ \step -> withFrozenCallStack $ do
-  fileContents <- ByteString.readFile ("semantic-python/test/fixtures" </> fp)
+fixtureTestTreeForFile :: HasCallStack => Path.RelFile -> Tasty.TestTree
+fixtureTestTreeForFile fp = HUnit.testCaseSteps (Path.toString fp) $ \step -> withFrozenCallStack $ do
+  let fullPath = Path.relDir "semantic-python/test/fixtures" </> fp
+
+  fileContents <- ByteString.readFile (Path.toString fullPath)
   directives <- case Directive.parseDirectives fileContents of
     Right dir -> pure dir
     Left err  -> HUnit.assertFailure ("Directive parsing error: " <> err)
 
   result <- TS.parseByteString TSP.tree_sitter_python fileContents
-  let coreResult = fmap (Control.Effect.run . runFail . Py.compile @(TSP.Module TS.Span) @_ @(Term (Ann :+: Core))) result
+  let coreResult = fmap (Control.Effect.run
+                          . runFail
+                          . runReader fp
+                          . Py.compile @(TSP.Module TS.Span) @_ @(Term (Ann :+: Core))) result
```

Comment:
I think this might express slightly more naturally with `<$>` instead of `fmap`.